pkg/jsonpatch/jsonpatch.go

```diff
@@ -0,0 +1,29 @@
+package jsonpatch
+
+import (
+	"bytes"
+
+	"gomodules.xyz/jsonpatch/v2"
+)
+
+type Operations []jsonpatch.JsonPatchOperation
+
+func (o Operations) MarshallJSON() ([]byte, error) {
```

Comment:
It was definitely a typo; fixing it. Re: `json.Marshal`, I thought about this approach but then decided to leverage the underlying `Operation.MarshalJSON` func since it provides some validation on fields; see [here](https://github.com/gomodules/jsonpatch/blob/93a4dafa4292a7e6cfff7d5d9aefc0f48219f07d/v2/jsonpatch.go#L26-L42).

Wrapping that existing func ensures the collection of patches are valid. Seems reasonable, no? 🤷🏼‍♂️

Reply:
Spoke offline - since `Operation` implements `Marshaler`, it's possible to just call `json.Marshal`.